src/pages/Embeds/Plio.vue

```diff
@@ -84,12 +98,19 @@ const PLYR_INTERVAL_TIME = 0.05;
 const UPLOAD_INTERVAL = 10000;
 var UPLOAD_INTERVAL_TIMEOUT = null;
 
+/**
+ * How many seconds before the end of the video
+ * should the scorecard pop up
+ */
+const SCORECARD_POPUP_TIME_FROM_END = 0.5;
```

Comment:
a slight confusion point is that the check for scoreboard pop-up uses 1 second and this uses 0.5 second.